public/js/index.js

```diff
@@ -1660,6 +1659,7 @@ function initWipTitle() {
         e.preventDefault();
 
         var $issueTitle = $("#issue_title");
+        $issueTitle.focus();
```

Comment:
It would be better to focus field after value has been set

Reply:
If I do that, then the cursor will be at the beginning of the input. By focusing before changing the content of the field, I make sure that my cursor will be at the end of it, which is preferable i think.

Reply:
Yes it should be at the end. For me it worked also if focused after value is set but ok, NVM than :)